From 7774520e0a11338fb3ee0f6b1c6d4f16f8327182 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 30 Apr 2019 18:23:29 +0300
Subject: [PATCH] [SW-0] pll: move goya specific offset to goya.h

This is the last remnant of Goya PLL base macro in pll.h.
Move it to goya.h.

Change-Id: I1cd8d358475b80d8cd7c566ee5cac3a5bf603194
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.h b/common/pll.h
index 4cce062..ca260c4 100644
--- a/common/pll.h
+++ b/common/pll.h
@@ -30,8 +30,6 @@
 #define BYPASS_CYCLES_PER_USEC	(SYS_REF_CLOCK / USEC_PER_SEC)
 #define MIN_CYCLES_PER_ITER	5
 
-#define mmPSOC_MME_PLL_BASE	0xC71000
-
 enum hl_pll_bypass_val {
 	HL_PLL_REF_CLK,
 	HL_PLL_PLL_CLK,
diff --git a/goya/goya.h b/goya/goya.h
index 40c36c0..0b1b996 100644
--- a/goya/goya.h
+++ b/goya/goya.h
@@ -7,3 +7,4 @@
 
 #define mmCPU_PLL_BASE		0x4A2000
 #define mmIC_PLL_BASE		0x4A3000
+#define mmPSOC_MME_PLL_BASE	0xC71000
